connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala

```diff
@@ -344,12 +345,13 @@ object KustoDataSourceUtils {
       ""
     }
 
-    logInfo("parseSinkParameters", s"Parsed write options for sink: {'timeout': '${writeOptions.timeout}, 'async': ${writeOptions.isAsync}, " +
+    logInfo("parseSinkParameters", s"Parsed write options for sink: {'table': '${sourceParameters.
+      kustoCoordinates.table}', 'timeout': '${writeOptions.timeout}, 'async': ${writeOptions.isAsync}, " +
       s"'tableCreationMode': ${writeOptions.tableCreateOptions}, 'writeLimit': ${writeOptions.writeResultLimit}, 'batchLimit': ${writeOptions.batchLimit}" +
       s", 'timeout': ${writeOptions.timeout}, 'timezone': ${writeOptions.timeZone}, " +
       s"'ingestionProperties': $ingestionPropertiesAsJson, 'requestId': '${sourceParameters.requestId}', 'pollingOnDriver': ${writeOptions.pollingOnDriver}," +
       s"'maxRetriesOnMoveExtents':$maxRetriesOnMoveExtents, 'minimalExtentsCountForSplitMergePerNode':$minimalExtentsCountForSplitMergePerNode, " +
-      s"'adjustSchema': $adjustSchema, 'autoCleanupTime': $autoCleanupTime${tempTableLog}")
+      s"'adjustSchema': $adjustSchema, 'autoCleanupTime': $autoCleanupTime}$tempTableLog")
```

Comment:
Shouldn't "$autoCleanupTime}$tempTableLog" be "$autoCleanupTime$tempTableLog}"?

Reply:
yes